src/components/StateManagement.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {stateLoad} from './../actions';
+
+const encoding = 'UTF-8';
+
+class StateManagement extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: props.state,
+      download: null
+    };
+    this.save = this.save.bind(this);
+    this.load = this.load.bind(this);
+    this.reset = this.reset.bind(this);
+  }
+
+  render() {
+    const {download} = this.state;
+    return <div>
+      {download
+        ? <a href={download} onClick={this.reset} download="initiate.json">Download</a>
+        : <button onClick={this.save}>Save</button>}
+      <input type="file" onChange={this.load}/>
+    </div>
+  }
+
+  save() {
+    const json = JSON.stringify(this.state.data);
+    const blob = new Blob([json], {encoding, type: `text/json;charset=${encoding}`});
```

Comment:
For prototype purposes I don't think a [file size limit of 500-800 MiB](https://github.com/eligrey/FileSaver.js/#supported-browsers) should be a problem.